addons/website/static/src/snippets/s_dynamic_snippet/000.js

```diff
@@ -249,14 +243,6 @@ const DynamicSnippet = publicWidget.Widget.extend({
     _toggleVisibility: function (visible) {
         this.$el.toggleClass('o_dynamic_snippet_empty', !visible);
     },
-    /**
-     * Returns the main URL of the module related to the active filter.
```

Comment:
@xO-Tx I would like to understand this... Which filter are we talking about?

Reply:
Discussed on Discord :+1: